### PR TITLE
Updated STXS analysis (work in progress)

### PR DESCRIPTION
**Pull request by edjtscott**
This is the latest version of the STXS analysis, with updated VH MET and tHq tags. There are also minor changes to the weight handling, and a source of uncertainty for the HEM issue is added. 

This branch has not been fully tested, and the ggH multiclassifier is not included yet. So please do not merge until further notice.

### CODE REVIEW

**Comment by simonepigazzini** [comment]
still in progress?

**Comment by edjtscott** [comment]
yes I have not got round to adding the ggH multiclassifier producer yet... will try to do so this week. 

Did @fabio-mon have any luck testing out the workflow including this?

**Comment by fabio-mon** [comment]
Not yet, hopefully this week.

**Comment by edjtscott** [comment]
hi @simonepigazzini - I fixed the conflict and added the latest changes (but not Lata's PR - I included that in the results but not here). Since I am going away tomorrow until 27th, it's probably best to just merge this (ideally someone can quickly check it first). I can then add the ggH multiclassifier later on, since it has a tiny effect on the workflow. Let me know what you think, cheers.

File: DataFormats/src/DiPhotonTagBase.cc [hunk 1]
```diff
@@ -76,13 +76,25 @@ void DiPhotonTagBase::setIsGold( int runNumber ) {
...
 
+// theory weights (in TagTruthBase) calculated standalone without diphoton object weight applied, so multiply it in here
 float DiPhotonTagBase::getTheoryWeight(std::string key) const
 {
     float theoryWeight = truth_->weight(key);
     theoryWeight *= this->centralWeight();
     return theoryWeight;
 }
 
+// overrides default behaviour of returning 1 if weight doesn't exist to return central object weight instead
```
Comment (simonepigazzini) [inline DataFormats/src/DiPhotonTagBase.cc:87]: @chernyavskaya heads up

**Pull request merged**